lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -155,6 +175,7 @@ div.course-wrapper {
       .gated-sequence {
         color: #147ABA;
         font-weight: 600;
+        padding: 16px 7px;
```

Comment:
Any way we can use denominations of our `$baseline` variable, which is 20px? Example: `padding: ($baseline / 1.5) ($baseline / 4);` which would equal (15px 5px).